python/unityagents/environment.py

```diff
@@ -87,6 +89,7 @@ def __init__(self, file_name, worker_id=0,
                                             "Provided filename does not match any environments."
                                             .format(true_filename))
         else:
+            logger.info("This is the launch string {}".format(launch_string))
             # Launch Unity environment
```

Comment:
Will the developer need to know the launch string ?

Reply:
This can be turned into a debug message.